sdk/samples/iot/docs/how_to_iot_hub_samples_linux.md

```diff
@@ -2,6 +2,8 @@
 
 ## Introduction
 
+Prerequisites:
```

Comment:
Why was this added?  There is a Prerequisites section below, and this is the Intro section.

Reply:
I have reorganized the code to move the prereq section to a header2 level (more noticeable)